src/Mod/TechDraw/Gui/Workbench.cpp

```diff
@@ -80,14 +80,53 @@ Gui::MenuItem* Workbench::setupMenuBar() const
     *dimensions << "TechDraw_LinkDimension";
     *dimensions << "TechDraw_LandmarkDimension";
 
-    // toolattributes
+    // extension: attributes and modifications
     Gui::MenuItem* toolattrib = new Gui::MenuItem;
-    toolattrib->setCommand("Extensions: centerlines and threading");
-    *toolattrib << "TechDraw_ExtensionCircleCenterLines";
-    *toolattrib << "TechDraw_ExtensionThreadHoleSide";
-    *toolattrib << "TechDraw_ExtensionThreadBoltSide";
-    *toolattrib << "TechDraw_ExtensionThreadHoleBottom";
-    *toolattrib << "TechDraw_ExtensionThreadBoltBottom";
+    toolattrib->setCommand("Extensions: attributes and modifications");
+    *toolattrib << "TechDraw_ExtensionSelectLineAttributes";
+    *toolattrib << "TechDraw_ExtensionExtendLine";
+    *toolattrib << "TechDraw_ExtensionShortenLine";
+    *toolattrib << "TechDraw_ExtensionLockUnlockView";
+    *toolattrib << "TechDraw_ExtensionPositionSectionView";
+    *toolattrib << "TechDraw_ExtensionChangeLineAttributes";
+    *toolattrib << "TechDraw_ExtensionPosHorizChainDimension";
+    *toolattrib << "TechDraw_ExtensionPosVertChainDimension";
+    *toolattrib << "TechDraw_ExtensionPosObliqueChainDimension";
+    *toolattrib << "TechDraw_ExtensionCascadeHorizDimension";
+    *toolattrib << "TechDraw_ExtensionCascadeVertDimension";
+    *toolattrib << "TechDraw_ExtensionCascadeObliqueDimension";
+
+    // extension: centerlines and threading
+    Gui::MenuItem* toolcenter = new Gui::MenuItem;
+    toolcenter->setCommand("Extensions: centerlines and threading");

```

Comment:
Please have a look at https://github.com/FreeCAD/FreeCAD/pull/5162

We also need to do the same for the newly introduced features of  `Extensions: dimensions` and `Extensions: attributes and modifications`